pkg/components/enterprise.go

```diff
@@ -15,135 +15,158 @@
 		Image:   "tigera/es-proxy",
 	}
-
+	
+	
 	ComponentFluentd = component{
-		Version: "v2.7.0",
-		Digest:  "sha256:b3921b735d220e10808362925e04500a19b1edc15a2c0108e5104a8317f51c52",
+		Version: "v2.7.0-0.dev-11-ge26adbb",
+		Digest:  "sha256:1ab8c0a65fc7a5f34af1fed90811938b1794a19866ab28ee032361b22fb6c03b",
 		Image:   "tigera/fluentd",
 	}
-
+	
+	
 	ComponentGuardian = component{
-		Version: "v2.7.0-0.dev-50-g2b3fa78",
-		Digest:  "sha256:22928d9b9711004c65699eb6daff547ce92cd6bd31feb680faff58d0a2a06d24",
+		Version: "v2.7.0-0.dev-61-g73e2b02",
+		Digest:  "sha256:707e77225e8b92a040c03a730cbcaa39c3109bf9387d30e253c4dbb10de7c42f",
 		Image:   "tigera/guardian",
 	}
-
+	
+	
 	ComponentIntrusionDetectionController = component{
-		Version: "v2.7.0-0.dev-38-gbf021d6",
-		Digest:  "sha256:11047f506a1db4d5295c1bd94afbdd2682c81cda9b9fccaafca4730f813bd4ee",
+		Version: "v2.7.0-0.dev-63-g8591567",
+		Digest:  "sha256:774a77e54ae42ed96aad05d910acaa7ee6872001ceca0eef3bb3e5305bf89b34",
 		Image:   "tigera/intrusion-detection-controller",
 	}
-
+	
+	
 	ComponentKibana = component{
-		Version: "7.3",
-		Digest:  "sha256:4c8e458fa0327c477c9aabd3672cc294e4497173faa61c475c76c47748677bf0",
+		Version: "v2.7.3-0-g99fc114",
+		Digest:  "sha256:ffc7704120c22f3d48d1495d205901795f16a51219fc79a01068dcbcde8cf33f",
 		Image:   "tigera/kibana",
 	}
-
+	
+	
 	ComponentManager = component{
-		Version: "v2.7.0",
-		Digest:  "sha256:439666b2f313341330ccece00cceff97b0c9cd77b4d834470060c519e51fccfc",
+		Version: "v2.7.0-0.dev-598-g5f43270b",
+		Digest:  "sha256:24fbd97411f81cd530ee749c42916d15698744b7664f3e9f7e12768c995a9008",
 		Image:   "tigera/cnx-manager",
 	}
-
+	
+	
 	ComponentManagerProxy = component{
-		Version: "v2.7.0-0.dev-50-g2b3fa78",
-		Digest:  "sha256:6ba6db0df465c06f64d1b241ed3b6e9bfa7a108a37851d45d13943d0bdbb6cff",
+		Version: "v2.7.0-0.dev-61-g73e2b02",
+		Digest:  "sha256:723b584f03e8b81e98bcd6002a9c599b3d31342edffe07a4c28bb01fae9a0ffb",
 		Image:   "tigera/voltron",
 	}
-
+	
+	
 	ComponentQueryServer = component{
-		Version: "v2.7.0",
-		Digest:  "sha256:3603e45257693d8112fee4244ed5ceeb09c6dd34df07eb5ddb1b2e882eb209ae",
+		Version: "v2.7.0-0.dev-74-g6ce4ce4",
+		Digest:  "sha256:06faecae3036ac48ec7e049ff596acea413f3da8523ad319d564e318cf04d1ca",
 		Image:   "tigera/cnx-queryserver",
 	}
-
+	
+	
 	ComponentTigeraKubeControllers = component{
-		Version: "v2.7.0-0.dev-153-g89e6dfa",
-		Digest:  "sha256:f0d152684829e26a91b846699e2301b7292f1e80684d290e5cd4ccd56d597e9e",
+		Version: "v2.7.0-0.dev-176-g68be5b8",
+		Digest:  "sha256:8f7400037804f3ae80168df50e6479ffba4998662f089984f3a5a92fe8b775a8",
 		Image:   "tigera/kube-controllers",
 	}
-
+	
+	
 	ComponentTigeraNode = component{
-		Version: "v2.7.0",
-		Digest:  "sha256:205c5e15543ef4c0f166a77326a061f5d99a1e20f6c9e4a0a8227b620333d280",
+		Version: "v2.7.0-0.dev-300-ge5619d5",
+		Digest:  "sha256:2b7434aedd81e84b9dccc14608f0af099c4f1e6926375b8ca0c8b646934ad064",
 		Image:   "tigera/cnx-node",
 	}
-
+	
+	
```

Comment:
Odd that these newlines were added. Perhaps you opened it in your editor and it applied formatting? We should really run the output of the template through `gofmt`. But fine as is